services/workflows-service/scripts/alerts/generate-alerts.ts

```diff
@@ -205,28 +156,87 @@ export const generateFakeAlertDefinition = async (
           name: faker.lorem.words(3),
           enabled: faker.datatype.boolean(),
           description: faker.lorem.sentence(),
-          rulesetId: `set-${rule.id}`,
+          rulesetId: `set-${inlineRule.id}`,
           defaultSeverity,
-          ruleId: rule.id,
+          ruleId: inlineRule.id,
           createdBy: createdBy,
           modifiedBy: createdBy,
           dedupeStrategies: {
             strategy: {},
             cooldownTimeframeInMinutes: faker.datatype.number({ min: 60, max: 3600 }),
           },
           config: { config: {} },
-          inlineRule: rule,
+          inlineRule,
           tags: [faker.helpers.arrayElement(tags), faker.helpers.arrayElement(tags)],
           additionalInfo: {},
           projectId: project.id,
-          alert: {
-            createMany: {
-              data: alerts,
-              skipDuplicates: true,
-            },
-          },
         },
-      });
-    }),
+      }),
+    ),
+  );
+
+const generateFakeAlert = ({
+  severity,
+  ids,
+}: {
+  severity: AlertSeverity;
+  ids: string[];
+}): Omit<Prisma.AlertCreateManyAlertDefinitionInput, 'projectId'> => {
+  const businessIdCounterpartyOriginatorIdOrBoth = faker.helpers.arrayElement(
+    ids.map(id => ({ counterpartyId: id })),
+  );
+
+  return {
+    dataTimestamp: faker.date.past(),
+    state: faker.helpers.arrayElement(Object.values(AlertState)),
+    status: faker.helpers.arrayElement(Object.values(AlertStatus)),
+    tags: [faker.random.word(), faker.random.word(), faker.random.word()],
+    executionDetails: JSON.parse(faker.datatype.json()) as InputJsonValue,
+    severity,
+    ...businessIdCounterpartyOriginatorIdOrBoth,
+    // TODO: Assign assigneeId value to a valid user id
+    // TODO: Assign counterpart value to a valid user id
+    // businessId: faker.datatype.uuid(),
+    // endUserId: faker.datatype.uuid(),
+    // assigneeId: faker.datatype.uuid(),
+    // workflowRuntimeDataId: faker.datatype.uuid()
```

Comment:
Our of scope, Can you improve seed and generate some counterparties as busniess and endUser ?